x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -223,23 +222,13 @@ export class ManifestManager {
     osOptions: BuildArtifactsForOsOptions
   ): Promise<Record<string, InternalArtifactCompleteSchema[]>> {
     const policySpecificArtifacts: Record<string, InternalArtifactCompleteSchema[]> = {};
-    await pMap(
-      allPolicyIds,
-      async (policyId) => {
-        for (const os of supportedOSs) {
-          policySpecificArtifacts[policyId] = policySpecificArtifacts[policyId] || [];
-          policySpecificArtifacts[policyId].push(
-            await this.buildArtifactsForOs({ os, policyId, ...osOptions })
-          );
-        }
-      },
-      {
-        concurrency: 5,
-        /** When set to false, instead of stopping when a promise rejects, it will wait for all the promises to
-         * settle and then reject with an aggregated error containing all the errors from the rejected promises. */
-        stopOnError: false,
+    for (const policyId of allPolicyIds)
```

Comment:
Do you know why we used pMap in the first place? The concurrency 5 value was for any particular reason?

Reply:
The purpose of that was to decrease running time. 5 was the default value we used before in the clean function of the manifest manager, nothing specific. But since the API call responses are now cached, there is no need to parallelise this I guess. Also, it could end up doing extra API calls if the cache is not filled yet and two parallel executions try to fill it.